### PR TITLE
feat: logging about downloadsnapshot failed

### PR DESCRIPTION
**Pull request by riemannulus**

### CODE REVIEW

**Review by boscohyun — APPROVED** [review]

File: src/main/snapshot.ts [hunk 1]
```diff
@@ -191,6 +191,7 @@ export async function downloadSnapshot(
     if (token.reason === "clear-cache") {
       throw new ClearCacheException();
     } else {
+      console.log(JSON.stringify(error));
```
Comment (longfin) [inline src/main/snapshot.ts:194]: Is `error`  encoded as JSON?

Comment (riemannulus) [reply src/main/snapshot.ts:194]: Yes. [(ref)](https://stackoverflow.com/a/18391400)

Comment (riemannulus) [reply src/main/snapshot.ts]: fixed: e353296

**Comment by riemannulus** [comment]
PTAL @longfin

**Review by longfin — APPROVED** [review]

**Pull request merged**